src/test/resources/application.yml

```diff
@@ -72,7 +72,7 @@ checkmarx:
   #exclude-folders:
 
 jira:
-  url: https://custodela.atlassian.net
+  url: http://localhost:8080
```

Comment:
Actually it might, but we should anyway avoid writing real URLs in our YAML, so I guess we can keep it that way.
As far as I understood from Aviv, we will create some mechanism that will put the right tokens, usernames, passowords and URLs as needed for the tests.